hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -898,15 +898,16 @@
             range clock option via a flag, and running several existing chip-level tests with that
             option, e.g.
 
-              chip_sw_clkmgr_jitter
-              chip_sw_flash_ctrl_ops_jitter_en
-              chip_sw_flash_ctrl_access_jitter_en
-              chip_sw_otbn_ecdsa_op_irq_jitter_en
-              chip_sw_aes_enc_jitter_en
-              chip_sw_hmac_enc_jitter_en
-              chip_sw_keymgr_key_derivation_jitter_en
-              chip_sw_kmac_mode_kmac_jitter_en
-              chip_sw_sram_ctrl_main_scrambled_access_jitter_en
+              chip_sw_clkmgr_jitter_reduced_freq
+              chip_sw_flash_ctrl_ops_jitter_en_reduced_freq
+              chip_sw_flash_ctrl_access_jitter_en_reduced_freq
+              chip_sw_otbn_ecdsa_op_irq_jitter_en_reduced_freq
+              chip_sw_aes_enc_jitter_en_reduced_freq
+              chip_sw_hmac_enc_jitter_en_reduced_freq
+              chip_sw_keymgr_key_derivation_jitter_en_reduced_freq
+              chip_sw_kmac_mode_kmac_jitter_en_reduced_freq
+              chip_sw_sram_ctrl_main_scrambled_access_jitter_en_reduced_freq
+              chip_sw_flash_init_reduced_freq
```

Comment:
These need to go in the `tests: []` list on line 913.

Reply:
Done.